app/liquid/liquid_renderer.rb

```diff
@@ -134,6 +134,7 @@ def isolate_from_plugin_data(field)
 
   def location
     return @location if @location.blank?
+    return {} if @location.data['country_code'] == 'RD'
```

Comment:
Good catch Neal. What's kind of messy of this code is that we're only overriding the `country` field from the member's stored data, and we leave the rest of the fields as they are, which may leave a pretty inconsistent location object. For example:

```
"country" => "Argentina" #Overriden by member's data
"country_name"=>"Reserved",
"country_code"=>"RD"
"ip"=>"127.0.0.1", 
"city"=>"",
 "region_code"=>"", 
"region_name"=>"",
"zipcode"=>"", 
"latitude"=>"0",
"longitude"=>"0", 
```